src/modules/Dropdown/Dropdown.js

```diff
@@ -755,6 +756,38 @@ export default class Dropdown extends Component {
     return _.findIndex(options, ['value', value])
   }
 
+  getDropdownAriaOptions = (ElementType) => {
+    const { loading, disabled, search, multiple } = this.props
+    const { open } = this.state
+    const ariaOptions = {
+      role: 'listbox',
+      'aria-busy': loading,
+      'aria-disabled': disabled,
+      'aria-expanded': !!open,
+    }
+    if (search) {
+      ariaOptions.role = 'combobox'
+    } else if (ElementType === Menu.Item) {
+      ariaOptions.role = 'menuitem'
```

Comment:
This role will be added automatically by the Menu.Item's aria props.  When augmenting components with the `as` prop, you essentially merging components.

I don't think the Dropdown should be aware of other components in this way.  Let's pull this one.